### DISCUSSION TITLE
Deploy Not Working

### DISCUSSION THREAD

**Discussion by iambherulal** [vercel/vercel#5033] [Help] [answered]
The provided GitHub repository does not contain the requested branch or commit reference. Please ensure the repository is not empty.

**Comment by styfle** [comment, marked as answer]
Hi @iambherulal 

Please submit a [Support Request](https://vercel.com/support/request) and provide the github repo and commit that failed to deploy.

We'll take a look and get it sorted out, thanks!

**Reply by NotMyself** [reply]
Was there a resolution to this issue? I am hitting it as well.

**Reply by marcmaceira** [reply]
Has this been resolved?

**Comment by meenatt** [comment]
my repository  isn't empty
